src/hb-ot-color-colr-table.hh

```diff
@@ -90,6 +90,474 @@ struct BaseGlyphRecord
   DEFINE_SIZE_STATIC (6);
 };
 
+template <typename T>
+struct Variable
+{
+  bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
+
+  protected:
+  T      value;
+  VarIdx varIdx; /* Use 0xFFFFFFFF to indicate no variation. */
+  public:
+  DEFINE_SIZE_STATIC (4 + T::static_size);
+};
+
+template <typename T>
+struct NoVariable
+{
+  bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (c->check_struct (this));
+  }
+
+  T      value;
+  public:
+  DEFINE_SIZE_STATIC (T::static_size);
+};
+
+// Color structures
+
+// The ColorIndex alpha is multiplied into the alpha of the CPAL entry
+// (converted to float -- divide by 255) looked up using paletteIndex to
+// produce a final alpha.
+template <template<typename> typename V>
```

Comment:
Any reason why you have the template argument called `V` in some and `Var` in other places? I like the `Var` more. Please unify.